plugins/outputs/kinesis/kinesis_test.go

```diff
@@ -323,3 +670,16 @@ func (m *mockKinesisPutRecords) AssertRequests(
 		}
 	}
 }
+
+func serializeMetric(
+	t *testing.T,
+	serializer serializers.Serializer,
+	metric telegraf.Metric,
+) []byte {
+
+	data, err := serializer.Serialize(metric)
+	if err != nil {
+		t.Errorf("Should have serialized test metric: %s", err.Error())
+	}
```

Comment:
```suggestion
	require.NoError(t, err)
```
I'm not so sure if those two-lines are worth an own function...